src/helpers/webdavHelper.js

```diff
@@ -134,7 +134,7 @@ exports.propfind = function (path, userId, properties, folderDepth = '1') {
  * @param {string} user
  * @param depth
  */
-exports.getTrashBinElements = function (user, depth = 'infinity') {
+exports.getTrashBinElements = function (user, depth = '1') {
```

Comment:
Note: for "ordinary" trashbin checks this will be fine. The top-level item in the trashbin will be contained in the response.

If there are test scenarios that want to look deeper in the trashbin then we will have to also write a smarter function. For example, if a scenario deletes a folder `folderToDelete`, and the folder has files in it, then there might be steps that want to check the trashbin entries that are inside `folderToDelete`, and to restore an individual file or folder inside `folderToDelete`.

If there are no scenarios like that yet, then doing just this change should work (and we can add more code later)